docs/weave/basics/01-blockchain.md

```diff
@@ -10,16 +10,17 @@ A "blockchain" in the simplest sense is a chain of blocks. By chain, we mean eac
 
 If you are coming from working on typical databases, you can think of the blockchain as an immutable [transaction log](https://en.wikipedia.org/wiki/Transaction_log). If you have worked with [Event Sourcing](https://martinfowler.com/eaaDev/EventSourcing.html) you can consider a block as a set of events that can always be replayed to create a [materialized view](https://docs.microsoft.com/en-us/azure/architecture/patterns/materialized-view). Maybe you have a more theoretical background and recognize that a blockchain is a fault tolerant form of [state machine replication](https://en.wikipedia.org/wiki/State_machine_replication#Ordering_Inputs).
 
-The point is given a node knows a block is valid(more on that in [consensus](./consensus.html)), it can cryptographically prove it has the valid history of that block, and then replay that sequence of blocks to reproduce the current state. Many nodes performing this simultaneously create a [Byzantine Fault Tolerant](https://en.wikipedia.org/wiki/Byzantine_fault_tolerance) state machine. Since (most) computer programs can be mapped to state machines, we end up with an [unstoppable world computer](https://www.ethereum.org/).
+The point is to prove that a given node's knowledge of a block is valid (more on that in [consensus](./consensus.html)).
+Each node can cryptographically prove it by reproducing the valid history to that block, and by replaying that sequence of blocks the node reproduces the current state. Many nodes performing this simultaneously create a [Byzantine Fault Tolerant](https://en.wikipedia.org/wiki/Byzantine_fault_tolerance) state machine. Since (most) computer programs can be mapped to state machines, we end up with an [unstoppable world computer](https://www.ethereum.org/).
```

Comment:
```suggestion
Each node can cryptographically prove it by reproducing the valid history to that block, and by replaying that sequence of blocks the node reproduces the current state. Many nodes performing this simultaneously create a [Byzantine Fault-Tolerant](https://en.wikipedia.org/wiki/Byzantine_fault_tolerance) state machine. Since (most) computer programs can be mapped to state machines, we end up with an [unstoppable world computer](https://www.ethereum.org/).
```